resources/lib/jellyfin/core/api.py

```diff
@@ -334,6 +334,3 @@ def close_transcode(self, device_id):
         return self._delete("Videos/ActiveEncodings", params={
             'DeviceId': device_id
         })
-
-    def delete_item(self, item_id):
-        return self.items("/%s" % item_id, "DELETE")
```

Comment:
flake8 was kind enough to informing me that this was a duplicate definition.
You should be able to find the exact same code further up.